app/views/observer/_form_observations.html.erb

```diff
@@ -160,13 +165,37 @@
     </div>
   </div>
   <div class="col-xs-12 col-sm-6 max-width-text">
-    <div class="well well-sm">
+    <div class="well well-sm" id="notes_help">
       <div class="arrow-left hidden-xs"></div>
       <%= content_tag(:p, :form_observations_notes_help.t, class: "help-block", style: "padding-top: 0px; margin-top: 0") %>
       <%= render(partial: "shared/textilize_help") %>
     </div>
   </div>
 </div>
+
+<% inject_javascript_at_end %(
```

Comment:
Closes Joe's issue of the help text overlapping when a user resizes the notes field on the create observation page.  This was the best little hack I could figure out which just hides the help field when a user resizes the text box.